tests/lib/rules/no-global-regexp-flag-in-query.test.ts

```diff
@@ -78,6 +78,19 @@ ruleTester.run(RULE_NAME, rule, {
         const utils = render(<Component/>)
         utils.notAQuery(/hello/i)
     `,
+
+    // issue #565
+    {
+      code: `
```

Comment:
The other valid snippets only contain the code block.

Reply:
Ah yes, I left this way because I was using the `only` property for debugging. I'll change it later.